velox/exec/tests/RowNumberTest.cpp

```diff
@@ -14,19 +14,76 @@
  * limitations under the License.
  */
 #include "velox/common/file/FileSystems.h"
+#include "velox/common/memory/MallocAllocator.h"
 #include "velox/exec/PlanNodeStats.h"
+#include "velox/exec/SharedArbitrator.h"
 #include "velox/exec/tests/utils/AssertQueryBuilder.h"
+#include "velox/exec/tests/utils/HiveConnectorTestBase.h"
 #include "velox/exec/tests/utils/OperatorTestBase.h"
 #include "velox/exec/tests/utils/PlanBuilder.h"
 #include "velox/exec/tests/utils/TempDirectoryPath.h"
 
+using namespace facebook::velox::memory;
+
 namespace facebook::velox::exec::test {
 
 class RowNumberTest : public OperatorTestBase {
  protected:
-  RowNumberTest() {
+  const static int64_t KB = 1024L;
+  const static int64_t MB = 1024L * KB;
+  const static uint64_t kMemoryCapacity = 512 * MB;
+  const static uint64_t kMemoryPoolInitCapacity = 16 * MB;
+  const static uint64_t kMemoryPoolTransferCapacity = 8 * MB;
+
+  void SetUp() override {
     filesystems::registerLocalFileSystem();
+    if (!isRegisteredVectorSerde()) {
+      this->registerVectorSerde();
+    }
+    executor_ = std::make_unique<folly::CPUThreadPoolExecutor>(32);
   }
+
+  void setupMemory(
```

Comment:
I expect this logic will be repeated in many tests. Can we extract it into helper methods in the base class?

Reply:
Got it, I will fix this once the refactor PR created by @xiaoxmeng is merged.